sources/osgShadow/ShadowMap.js

```diff
@@ -902,8 +944,8 @@ ShadowMap.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( S
         mat4.copy( this._projectionMatrix, camera.getProjectionMatrix() );
         mat4.copy( this._viewMatrix, camera.getViewMatrix() );
 
-        this._texture.setViewMatrix( this._viewMatrix );
-        this._texture.setProjectionMatrix( this._projectionMatrix );
+        this._texture.setViewMatrix( this._viewMatrix, this._lightIndex );
```

Comment:
use the lightNum as first parameter, and we should only use the term lightNum and lightNumber is it's related to lightNumber in Light.js